MAUI/Calendar/migration.md

```diff
@@ -429,7 +429,7 @@ this.Content = calendar;
 <tr>
 <td>{{'[ShowHeader](https://help.syncfusion.com/cr/xamarin/Syncfusion.SfCalendar.XForms.SfCalendar.html#Syncfusion_SfCalendar_XForms_SfCalendar_ShowHeader)'| markdownify }}</td>
 <td>Nil</td>
-<td>Not Supported.</td>
+<td>By using [Height](https://help.syncfusion.com/maui/calendar/headers#customize-the-header-height) property of the HeaderView value as 0 in sample level you can hide and show the headers.</td>
```

Comment:
Instead of saying sample level you can directly update which way can achieve this support